app/models/case/base.rb

```diff
@@ -423,6 +423,35 @@ def responded_in_time?
     date_responded <= external_deadline
   end
 
+  # determines whether or not an individual BU responded to a case in time, measured
+  # from the date the case was assigned to the business unit to the time the case was marked as responded.
+  # Note that the time limit is different for trigger cases (the internal time limit) than for non trigger
+  # (the external time limit)
+  #
+  def business_unit_responded_in_time?
+    responding_transitions = transitions.where(event: 'respond')
```

Comment:
We don't have a 'respond' event for SARs anymore, the KILO just closes the case and enters their own date